lib/ex_webrtc/sdp_utils.ex

```diff
@@ -81,6 +81,16 @@ defmodule ExWebRTC.SDPUtils do
     end
   end
 
+  @spec ensure_rtcp_mux(ExSDP.t()) :: :ok | {:error, :rtcp_mux_not_indicated}
+  def ensure_rtcp_mux(sdp) do
+    sdp.media
+    |> Enum.all?(&(ExSDP.Media.get_attribute(&1, :rtcp_mux) == :rtcp_mux))
+    |> case do
+      true -> :ok
+      false -> {:error, :rtcp_mux_not_included}
```

Comment:
To be consistent with other functions 

```suggestion
      false -> {:error, :missing_rtcp_mux}
```